src/OrchardCore.Modules/OrchardCore.Users/DataMigrations/AssignRoleToUsersMigrations.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Data.Migration;
+using OrchardCore.Environment.Shell.Scope;
+using OrchardCore.Security;
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Users.DataMigrations;
+
+public sealed class AssignRoleToUsersMigrations : DataMigration
```

Comment:
This doesn't seem to be fully working.

1. Blog setup on `main`.
2. Set the two top permissions:
![image](https://github.com/user-attachments/assets/f0a732f7-5a0d-44fb-8079-6610ad050758)

3. Switch to issue branch.
4. Only Assign Administrator is retained:

![{8B0BF08B-A3B9-4377-B08A-D793616F59BD}](https://github.com/user-attachments/assets/404e6223-e52e-473e-911d-864308696b06)

Reply:
I’ll test it out tomorrow. At some point, I asked myself do we even need this migration since it’s note even used anywhere. But decided “it won’t hurt much to add it”.

Reply:
I don't think we should add this migration. Since the permissions weren't used at all, we might assign additional rights to the users during the migration. We shouldn't do this because it can have safety-critical side effects.

Reply:
It was a bug though, that the old permissions didn't work.

Reply:
I asked myself the same question and @gvkries confirmed my thoughts. There is not point of having the migration. 

> It was a bug though, that the old permissions didn't work.

But they also have not worked in a long time. so we are just going to obsolete them and document the change. If someone want to get the same behavior, they would just change the permission check on their end.

Reply:
in version 1.6 https://github.com/OrchardCMS/OrchardCore/commit/009daa30af536801583ec423a9f7d4c9ab7c2c70

Reply:
OK then, migration not needed.